wallet-new/bench/Client/Cardano/Wallet/Web/Run.hs

```diff
@@ -0,0 +1,60 @@
+-- | Run endpoint client.
+
+module Client.Cardano.Wallet.Web.Run
+    ( runEndpointClient
+    ) where
+
+import           Universum
+
+import           Servant.Client             (ClientM, ClientEnv (..), runClientM)
+import           Servant.Client.Core        (BaseUrl (..), Scheme (..))
+
+import           Data.Default               (def)
+import           Network.Connection         (TLSSettings (..))
+import           Network.HTTP.Client        (Manager, newManager)
+import           Network.HTTP.Client.TLS    (mkManagerSettings)
+import           Network.TLS                (ClientParams (..), credentialLoadX509FromMemory,
+                                             defaultParamsClient, onCertificateRequest,
+                                             onServerCertificate, supportedCiphers)
+import           Network.TLS.Extra.Cipher   (ciphersuite_all)
+
+import           Bench.Cardano.Wallet.Types (CompleteConfig (..))
+
+-- | Run client for particular endpoint. It is assumed that
+-- node is already running, with enabled Wallet Web API.
+runEndpointClient
+    :: CompleteConfig
+    -> ClientM a
+    -> IO (Either Text a)
+runEndpointClient CompleteConfig {..} realClient = do
+    manager <- makeClientManager tlsPubCert tlsPrivKey
+    runClientM realClient (ClientEnv manager nodeURL) >>= \case
+        Left problem   -> return . Left  $ toText (show problem :: String)
+        Right response -> return . Right $ response
+  where
+    nodeURL  = BaseUrl Https nodeHost nodePort ""
+    nodeHost = "localhost"
```

Comment:
Configurable?

Reply:
I don't know if it should be configurable. Benchmarking is always running on localhost, right?

Reply:
What about the port? Should that be configurable?